test/db-migrations/20250113-01-disable-nullable-blob-content-types.spec.js

```diff
@@ -0,0 +1,58 @@
+const assert = require('node:assert/strict');
+const { hash, randomBytes } = require('node:crypto');
+
+const { // eslint-disable-line object-curly-newline
+  assertTableContents,
+  describeMigration,
+  rowsExistFor,
+} = require('./utils'); // eslint-disable-line object-curly-newline
+
+describeMigration('20250113-01-disable-nullable-blob-content-types', ({ runMigrationBeingTested }) => {
+  const aBlobWith = props => {
+    const randomContent = randomBytes(100);
+    const md5 = hash('md5',  randomContent); // eslint-disable-line no-multi-spaces
+    const sha = hash('sha1', randomContent);
+    return { md5, sha, ...props };
+  };
+  const aBlob = () => aBlobWith({});
+
+  const blob1 = aBlobWith({ contentType: null });
+  const blob2 = aBlobWith({ contentType: 'text/plain' });
+
+  before(async () => {
+    await rowsExistFor('blobs', blob1, blob2);
+
+    await runMigrationBeingTested();
+  });
+
+  it('should change existing NULL contentType values to application/octet-stream, and preserve non-NULL values', async () => {
+    await assertTableContents('blobs',
+      { ...blob1, contentType: 'application/octet-stream' },
+      { ...blob2, contentType: 'text/plain' },
+    );
+  });
+
+  it(`should create new blobs with contentType 'application/octet-stream' (contentType not supplied)`, async () => {
+    const { md5, sha } = aBlob();
+
+    const created = await db.oneFirst(sql`
+      INSERT INTO blobs (md5, sha, "contentType")
```

Comment:
This test appears to be the same as the following one.  Should `"contentType"` be removed?